2020-05-13 08:34:54.8508|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:34:55.9449|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:34:56.1548|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:34:56.3025|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:35:04.9304|ERROR|WedigITCRM.Maintenance.SynchronizeDineroContactsService|Forced error in SynchronizeDineroContactsService
2020-05-13 08:37:04.9705|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:37:05.3899|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:37:05.4807|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:37:05.5676|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:37:12.5680|ERROR|WedigITCRM.Maintenance.SynchronizeDineroContactsService|Forced error in SynchronizeDineroContactsService
2020-05-13 08:38:10.1826|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:38:10.4996|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:38:10.5687|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:38:10.6522|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:38:14.4239|ERROR|WedigITCRM.Maintenance.SynchronizeDineroContactsService|Forced error in SynchronizeDineroContactsService
2020-05-13 08:41:15.4304|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:41:15.9428|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:41:16.0550|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:41:16.1719|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:42:30.9592|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:42:31.6313|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:42:32.1886|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:42:32.3524|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:43:56.5898|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:43:57.2793|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:43:57.3873|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:43:57.4968|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:47:52.2280|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:47:54.9704|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:47:55.1333|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:47:55.3564|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:49:37.2614|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:49:40.1480|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:49:40.2262|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:49:40.3094|ERROR|WedigITCRM.Startup|Endpoint Routing does not support 'IApplicationBuilder.UseMvc(...)'. To use 'IApplicationBuilder.UseMvc' set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices(...).
2020-05-13 08:55:44.4121|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 08:55:46.6316|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 08:55:46.7731|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 08:55:47.1721|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 08:55:54.9090|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 08:55:54.9693|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to frontpage controller index action
2020-05-13 08:56:43.4697|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 08:56:43.4697|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 09:27:47.2611|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 09:27:49.2100|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 09:27:49.3378|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 09:27:49.5952|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 09:28:02.2798|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 09:28:02.2798|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 09:49:40.1859|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 09:49:41.5916|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 09:49:41.7435|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 09:49:42.3944|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 09:49:53.0504|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 09:49:53.0596|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 09:57:02.3361|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 09:57:02.5732|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 09:57:56.4212|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 09:57:56.5872|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 10:00:40.2686|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 10:00:41.7319|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 10:00:42.0116|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 10:00:42.4909|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 10:00:48.1852|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 10:00:48.1852|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 10:25:57.9938|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 10:25:59.6135|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 10:25:59.7184|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 10:25:59.9269|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 10:26:00.7574|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 10:26:00.7574|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to frontpage controller index action
2020-05-13 10:27:08.4472|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 10:27:08.4557|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to frontpage controller index action
2020-05-13 10:27:19.3302|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 10:27:19.3302|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 17:03:12.9447|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 17:03:14.1000|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 17:03:14.3137|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 17:03:14.6409|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 17:03:15.6318|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 17:03:15.6318|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to frontpage controller index action
2020-05-13 17:03:33.8165|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 17:03:33.8312|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 17:55:07.0904|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 17:55:08.6889|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 17:55:08.8733|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 17:55:09.4207|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 17:55:12.9636|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 17:55:12.9701|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 18:17:23.7337|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 18:17:24.5846|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 18:17:24.6957|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 18:17:24.9521|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 18:17:28.4485|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 18:17:28.4636|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 18:39:25.8686|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 18:39:26.0858|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 18:39:54.2263|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 18:39:54.3585|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 18:40:13.7734|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 18:40:15.6145|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 18:40:15.7842|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 18:40:16.2229|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 18:40:20.3654|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 18:40:20.3774|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 18:49:41.8348|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 18:49:41.8626|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"The remote server returned an error: (404) Not Found.\r\n                                         \r\n\r\n                                            at System.Net.HttpWebRequest.GetResponse()\r\n   at System.Net.WebClient.GetWebResponse(WebRequest request)\r\n   at System.Net.WebClient.DownloadBits(WebRequest request, Stream writeStream)\r\n   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)\r\n   at System.Net.WebClient.DownloadString(Uri address)\r\n   at System.Net.WebClient.DownloadString(String address)\r\n   at WedigITCRM.DineroAPI.DineroContacts.getContactFromDinero(Guid dineroId) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 51\r\n   at WedigITCRM.DineroAPI.DineroContacts.UpdateContactInDinero(Company contactCompanyOrPerson, Guid dineroContactGuid) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 95\r\n   at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 161\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"e94d1b90-36fa-4603-9fb6-a7f6374bacb6","Extensions":{}}
2020-05-13 18:56:36.1193|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 18:56:36.7786|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 18:56:36.8603|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 18:56:37.1035|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 18:56:40.1765|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 18:56:40.1913|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 18:57:36.0637|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 18:57:36.3379|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"Object reference not set to an instance of an object.\r\n                                         \r\n\r\n                                            at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 252\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"908f2424-2f59-4a52-90ec-26ee4b088d1f","Extensions":{}}
2020-05-13 19:00:12.3245|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 19:00:12.3503|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"The remote server returned an error: (404) Not Found.\r\n                                         \r\n\r\n                                            at System.Net.HttpWebRequest.GetResponse()\r\n   at System.Net.WebClient.GetWebResponse(WebRequest request)\r\n   at System.Net.WebClient.DownloadBits(WebRequest request, Stream writeStream)\r\n   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)\r\n   at System.Net.WebClient.DownloadString(Uri address)\r\n   at System.Net.WebClient.DownloadString(String address)\r\n   at WedigITCRM.DineroAPI.DineroContacts.getContactFromDinero(Guid dineroId) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 51\r\n   at WedigITCRM.DineroAPI.DineroContacts.UpdateContactInDinero(Company contactCompanyOrPerson, Guid dineroContactGuid) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 95\r\n   at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 161\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"8d279b3e-aea5-4206-964c-84e771b49e1a","Extensions":{}}
2020-05-13 19:03:13.7325|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 19:03:13.7552|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"The remote server returned an error: (404) Not Found.\r\n                                         \r\n\r\n                                            at System.Net.HttpWebRequest.GetResponse()\r\n   at System.Net.WebClient.GetWebResponse(WebRequest request)\r\n   at System.Net.WebClient.DownloadBits(WebRequest request, Stream writeStream)\r\n   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)\r\n   at System.Net.WebClient.DownloadString(Uri address)\r\n   at System.Net.WebClient.DownloadString(String address)\r\n   at WedigITCRM.DineroAPI.DineroContacts.getContactFromDinero(Guid dineroId) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 51\r\n   at WedigITCRM.DineroAPI.DineroContacts.UpdateContactInDinero(Company contactCompanyOrPerson, Guid dineroContactGuid) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 95\r\n   at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 161\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"9c259c9c-7dfa-4ab1-b92b-a0fa091ac9fa","Extensions":{}}
2020-05-13 19:06:38.1092|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 19:06:38.1332|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"The remote server returned an error: (404) Not Found.\r\n                                         \r\n\r\n                                            at System.Net.HttpWebRequest.GetResponse()\r\n   at System.Net.WebClient.GetWebResponse(WebRequest request)\r\n   at System.Net.WebClient.DownloadBits(WebRequest request, Stream writeStream)\r\n   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)\r\n   at System.Net.WebClient.DownloadString(Uri address)\r\n   at System.Net.WebClient.DownloadString(String address)\r\n   at WedigITCRM.DineroAPI.DineroContacts.getContactFromDinero(Guid dineroId) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 51\r\n   at WedigITCRM.DineroAPI.DineroContacts.UpdateContactInDinero(Company contactCompanyOrPerson, Guid dineroContactGuid) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\DineroAPI\\DineroContacts.cs:line 95\r\n   at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 161\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"cff33210-376e-4a3e-be7f-33f7c72f3f41","Extensions":{}}
2020-05-13 19:08:19.5126|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 19:08:19.5218|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"Object reference not set to an instance of an object.\r\n                                         \r\n\r\n                                            at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 252\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"cd953fe7-f25a-4158-9779-2989cc0f0c96","Extensions":{}}
2020-05-13 19:08:39.6328|ERROR|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.
2020-05-13 19:08:39.6492|ERROR|WedigITCRM.Startup|{"Type":null,"Title":"Unexpected Error","Status":500,"Detail":"Object reference not set to an instance of an object.\r\n                                         \r\n\r\n                                            at WedigITCRM.Controllers.CustomerController.EditCompany(CompanyInputPostModel datamodelInput, CompanyAccount companyAccount) in C:\\Users\\John\\source\\repos\\WedigITCRM\\WedigITCRM\\Controllers\\CustomerController.cs:line 252\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)","Instance":"f8c252b7-c1ce-4bd9-8432-a08c41a4f1ae","Extensions":{}}
2020-05-13 19:10:02.4984|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 19:10:03.5268|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 19:10:03.7499|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 19:10:03.9282|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 19:10:07.1009|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 19:10:07.1122|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 19:18:03.9642|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 19:18:04.3766|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 19:18:04.4949|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 19:18:05.1439|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 19:18:08.9767|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 19:18:08.9877|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 20:05:13.2812|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 20:05:14.9499|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 20:05:15.0319|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 20:05:15.2671|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 20:05:37.0046|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 20:05:37.0172|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 20:12:45.3842|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 20:12:45.7885|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 20:12:45.8738|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 20:12:46.1414|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 20:12:53.6053|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 20:12:53.6150|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 20:49:12.3619|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 20:49:12.5978|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 20:49:48.8822|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 20:49:49.1170|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 20:50:16.5407|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 20:50:16.7712|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 20:51:54.7440|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 20:51:54.8541|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:23:57.4523|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:23:57.8783|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:31:48.4773|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:31:49.2718|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:35:32.6666|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:35:33.0102|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:36:13.5323|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:36:13.7189|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:50:05.8890|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:50:06.0871|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:51:09.3784|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:51:09.5309|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:52:41.8999|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:52:42.1205|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:53:10.2977|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:53:10.5131|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:54:12.5957|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:54:12.8031|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:55:56.9428|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:55:57.2628|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:56:40.4532|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:56:40.5985|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 21:58:42.6574|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 21:58:42.8222|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 22:34:32.0636|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 22:34:32.2563|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 22:36:07.9321|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 22:36:08.0416|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 22:41:32.2371|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 22:41:32.3710|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 22:43:08.3556|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 22:43:08.4804|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 22:44:19.9094|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 22:44:20.1720|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 22:44:55.7301|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 22:44:55.8430|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:22:36.0877|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:22:36.2539|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:23:10.3705|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:23:10.4850|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:23:32.7510|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:23:33.4416|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:23:33.5291|ERROR|WedigITCRM.Startup|start of Configure method
2020-05-13 23:23:33.7324|ERROR|WedigITCRM.Startup|end of Configure method
2020-05-13 23:23:34.4169|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 23:23:34.4227|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to frontpage controller index action
2020-05-13 23:23:47.9025|ERROR|WedigITCRM.Controllers.HomeController|Entering index action in home controller
2020-05-13 23:23:47.9025|ERROR|WedigITCRM.Controllers.HomeController|index action in home controller redirecting to Customer controller AllCustomers action
2020-05-13 23:51:03.3216|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:51:03.4896|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:52:10.5028|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:52:10.6178|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:54:03.3842|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:54:03.5020|ERROR|WedigITCRM.Startup|end of ConfigureServices
2020-05-13 23:55:46.0841|ERROR|WedigITCRM.Startup|start of ConfigureServices
2020-05-13 23:55:46.2390|ERROR|WedigITCRM.Startup|end of ConfigureServices
